corehq/apps/app_manager/models.py

```diff
@@ -5143,7 +5143,7 @@ def make_build(self, comment=None, user_id=None, previous_version=None):
         if not copy._id:
             # I expect this always to be the case
             # but check explicitly so as not to change the _id if it exists
-            copy._id = copy.get_db().server.next_uuid()
+            copy._id = uuid.uuid4().hex
```

Comment:
Thanks, that is interesting.

I just realized that this is a bit pointless because if you save a doc without an ID couchdbkit will still call `next_uuid` to get the doc ID. 

Either way, without forms and cases in couch we aren't generating THAT many new docs so I don't think this should be an issue